qa/integration-tests/src/test/java/io/camunda/it/utils/CamundaMultiDBExtension.java

```diff
@@ -137,12 +138,20 @@ public class CamundaMultiDBExtension
     implements AfterAllCallback, BeforeAllCallback, ParameterResolver {
   public static final String PROP_CAMUNDA_IT_DATABASE_TYPE =
       "test.integration.camunda.database.type";
+  public static final String TEST_INTEGRATION_OPENSEARCH_AWS_URL =
+      "test.integration.opensearch.aws.url";
+  public static final String PROP_TEST_INTEGRATION_OPENSEARCH_AWS_TIMEOUT =
+      "test.integration.opensearch.aws.timeout.seconds";
+  public static final Duration TIMEOUT_DATABASE_EXPORTER_READINESS_TIMEOUT =
```

Comment:
🔧 I would probably rename them if they are used in the tests to something like:

```suggestion
  public static final Duration TIMEOUT_DATA_AVAILABILITY =
```

This is also somehow duplicated TIMEOUT__TIMEOUT

Reply:
That's weird I didn't noticed it. Probably changed somewhere down refactoring 🙈 . Will rename it to as you suggested